### DISCUSSION TITLE
Whitelist filling with random gibberish

### DISCUSSION THREAD

**Discussion by ikaevalko** [nefarius/HidHide#35] [Q&A]
I have been experiencing some odd behavior regarding the whitelist functionality in HidHide. When I try to add an application to the list, it gets flooded by these weird lines with random characters and file paths. The worst thing is that attempting to delete these erroneous lines from the list just produces more of them! Reinstalling did not fix the issue.
![whitelist](https://user-images.githubusercontent.com/40829126/126030369-d5b55983-d8f8-4045-99c3-cfd0c8e2b619.png)

**Comment by nefarius** [comment]
Moved to the proper location since everything making a good **bug report** is missing

**Reply by jjensen6823** [reply]
I’m having exactly the same issue.  I even formatted the hard drive and reinstalled everything, to no avail.  Was there ever a resolution to this?

**Reply by nefarius** [reply]
Is there maybe anti-virus other than Windows Defender installed? What is your exact OS version and language used?

Cheers
